queue/example_test.go

```diff
@@ -1,3 +1,11 @@
+// Setup queue module and start Tarantool instance before execution:
+// Terminal 1:
+// $ make deps
+// $ TT_LISTEN=3013 tarantool queue/config.lua
```

Comment:
There is a `TEST_TNT_LISTEN` env var for all test config files

Reply:
Tarantool allows passing box.cfg options via `TT_<NAME>` env variables. Command-line provided in a comment works.

Reply:
Well, there is a reason to use custom variables instead of `TT_` ones, since we set LISTEN port not on the second `box.cfg{}` call, not the first one. It may have no effect in case of manual test instance startup, but I still do not see any reasons in using two type of variables to configure the instance: consistency is good.

Reply:
Replaced `TT_<NAME>` with `TEST_TNT_<NAME>`.